Geometry/CMSCommonData/python/cmsExtendedGeometry2017aXML_cfi.py

```diff
@@ -0,0 +1,287 @@
+import FWCore.ParameterSet.Config as cms
+
+## 2015 + new phase 1 pixel detector
+
+XMLIdealGeometryESSource = cms.ESSource("XMLIdealGeometryESSource",
```

Comment:
Could you, please, make sure that this new scenario differs from the default 2017 only in HCal? Thanks.